src/components/data/DataTable.js

```diff
@@ -480,7 +482,7 @@ const DataTable = (props) => {
                             field: 'name',
                             onSort: setSort,
                             renderSearch: !!googleProject,
-                            searchByColumn: (v) => searchByColumn(entityMetadata[entityType].idName, v),
+                            searchByColumn: (v) => searchByColumn(entityMetadata[entityType].idName, v, 'name'),
```

Comment:
It looks like the 3rd argument here is no longer needed.

Reply:
i know i saw the "bug". was letting circleci run since it failed the last couple times